timing/timings.1.csv

```diff
@@ -0,0 +1,62 @@
+size set 1,size set 2,execution time (s)
+50,50,2.724760055541992
+100,100,2.7670867443084717
+150,150,3.1778759956359863
+200,200,4.251028060913086
+250,250,4.484825134277344
+300,300,5.064546823501587
+350,350,6.005185127258301
+400,400,6.675647974014282
+450,450,8.158300161361694
+500,500,9.899688959121704
+550,550,11.776095867156982
+600,600,21.683282136917114
+650,650,26.079952716827393
+700,700,21.65829610824585
+750,750,19.618257999420166
+800,800,21.968334913253784
+850,850,24.83933997154236
+900,900,25.993783950805664
+950,950,29.35918092727661
+1000,1000,32.30607199668884
+1050,1050,35.57166624069214
+1100,1100,39.96245288848877
+1150,1150,44.65693402290344
+1200,1200,48.085132122039795
+1250,1250,48.37261986732483
+1300,1300,53.667048931121826
+1350,1350,58.95645308494568
+1400,1400,58.556304931640625
+1450,1450,61.71402716636658
+1500,1500,68.26699805259705
+1550,1550,71.41845297813416
+1600,1600,74.8698103427887
+1650,1650,79.21338891983032
+1700,1700,83.70906186103821
+1750,1750,89.62421011924744
+1800,1800,93.3082628250122
+1850,1850,98.36845397949219
+1900,1900,103.52221918106079
+1950,1950,108.15761089324951
+2000,2000,112.74910593032837
+2050,2050,120.03180313110352
+2100,2100,126.09542202949524
+2150,2150,132.66069197654724
+2200,2200,139.6946041584015
+2250,2250,150.98278617858887
+2300,2300,155.74986696243286
+2350,2350,165.3443899154663
+2400,2400,169.11304092407227
+2450,2450,178.14109706878662
+2500,2500,182.99004101753235
+2550,2550,192.63860201835632
+2600,2600,202.99907779693604
+2650,2650,209.94929814338684
+2700,2700,216.70873308181763
+2750,2750,220.71831011772156
+2800,2800,233.41998600959778
+2850,2850,235.86974000930786
```

Comment:
What's the difference between timings and timings.1?

Reply:
`Timings.1` contains measurements after I applied the first optimization, `timings` contains the measurements with both the first and second optimization enabled. I think I'll just delete `timings.1` as we still have these files in a Google spreadsheet.

Reply:
Could you link the google spreadsheet to the code somehow? Prabably an optimization issue is the most sensible place.

Reply:
Good idea, I added the link here: https://github.com/MEGA-GO/Mega-GO/issues/13#issuecomment-627795374